recipes/jsoncpp/all/conanfile.py

```diff
@@ -37,8 +37,8 @@ def _patch_sources(self):
 
         if self.settings.os != "Windows" and not self.options.shared and not self.options.fPIC:
             tools.replace_in_file(os.path.join(self._source_subfolder, "src", "lib_json", "CMakeLists.txt"),
-                                  "set_target_properties( jsoncpp_lib PROPERTIES POSITION_INDEPENDENT_CODE ON)",
-                                  "set_target_properties( jsoncpp_lib PROPERTIES POSITION_INDEPENDENT_CODE OFF)")
+                                  "POSITION_INDEPENDENT_CODE ON",
+                                  "POSITION_INDEPENDENT_CODE OFF")
```

Comment:
Perhaps this could be a patch (to remove it) for the two versions in questions in the `conandata.yml`

Reply:
If it's a better approach, than I can do it, I suppose.

Reply:
Let the CCI team answer that :stuck_out_tongue_closed_eyes:

Reply:
YOu can literally remove it from the cmake file, the conan cmake wrapper, using conan_basic_setup sets it globally. e.g. replace("set_target_properties( jsoncpp_lib PROPERTIES POSITION_INDEPENDENT_CODE ON)", "")

This recipe has many small patches, but them depends on the version, or an option, you can create patches of course, but requires more effort as you need to add new conditions in cmake. For this one, you keep `tools.replace_in_file`

Reply:
I made a replace, which basically deletes this options.
It depends on the version of JsonCpp, but there is no if for OS or something else

Reply:
Unfortunately, doesn't seem to work. So I falled back to my original version.